packages/sign-client/src/controllers/engine.ts

```diff
@@ -681,6 +683,9 @@ export class Engine extends IEngine {
         done(),
       ]);
     } else if (this.client.core.pairing.pairings.keys.includes(topic)) {
+      this.client.logger.warn(
+        "ping() on pairing topic is deprecated and will be removed in the next major release.",
```

Comment:
Should it say ping here?

Reply:
yes, since we have single `PING()` for both session & pairing topics, only when pairing is provided we log the warning